plugins/plugin-urlfactory/src/main/java/org/eclipse/che/plugin/urlfactory/URLFetcher.java

```diff
@@ -40,6 +42,9 @@
   /** Maximum size of allowed data. (30KB) */
   protected static final long MAXIMUM_READ_BYTES = 30 * 1000;
 
+  /** The Compiled REGEX PATTERN that can be used for http|https git urls */
+  final Pattern GIT_HTTP_URL_PATTERN = Pattern.compile("(?<sanitized>^http[s]?://.*)\\.git$");
```

Comment:
I believe that `s`  doesn't have to be in square brackets, so it can be removed to simplify regexp if you will.